### ISSUE TITLE
Error: Cannot set property 'dropArea' of undefined

### ISSUE THREAD

**Issue by jakemmarsh** [madebymany/sir-trevor-js#316]
I'm installing sir-trevor via npm and packaging via browserify. The relevant lines:

package.json: `"sir-trevor": "^0.5.0-beta3"` (as a dependency)
Editor.jsx (React component): `var SirTrevor    = require('sir-trevor');`

However, once I compile/run this code, I get the following error and stack trace:

```
/Users/jakemarsh/Repos/acadeME/node_modules/sir-trevor/src/helpers/event.js:27
$.fn.dropArea = function(){
              ^
TypeError: Cannot set property 'dropArea' of undefined
    at Object.<anonymous> (/Users/jakemarsh/Repos/acadeME/node_modules/sir-trevor/src/helpers/event.js:27:15)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/Users/jakemarsh/Repos/acadeME/node_modules/sir-trevor/src/index.js:10:1)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
Program node server.js exited with code 1
```

I'm not sure why this would be. Both my project and sir-trevor have jQuery as a dependency, and it's correctly required. I've tried using `console.log`, and confirmed that `$` is an object at the time this line is called. `$.fn`, however, is undefined.

Any ideas?

**Comment by zgryw** [comment]
For me it sounds like your `$` isn't really jQuery, but I'm gonna look at your project at get back to you.

**Comment by jakemmarsh** [comment]
@zgryw you were right about `$` not really being jQuery. My app is isomorphic, meaning it's run/rendered on both the server and client. This means that when jQuery was loaded on the server side, it wasn't returning the actual jQuery object since that requires `window` to exist.

I fixed it by moving `require('sir-trevor')` into my `componentDidMount` function call (ReactJS method).

However, I'm now running into this problem (possibly with the same root cause?):

```
Uncaught ReferenceError: define is not defined                            scribe.js:1
```

**Comment by zgryw** [comment]
Interesting. 

The reason for that error is that `scribe` uses AMD-style packages and to use them with browserify you have to use [deamdify](https://www.npmjs.com/package/deamdify) transformation. I'm surprised that just requiring sir trevor isn't enough though...

**Comment by ninjabiscuit** [comment]
I think this might be broken. I definately had this working the other day without the need for a top level deamdify. That should be taken care of when the file is compiled. If it's not currently working that way then it should do

**Comment by ninjabiscuit** [comment]
Issue #327 describes the scribe issue in more detail and I'm going to close this issue in favour of that one. @jakemmarsh If you'd like to stay up to date with a fix for this, maybe track #327 .
